operator/v1/types_network.go

```diff
@@ -459,6 +459,13 @@ type OVNKubernetesConfig struct {
 	// egressIPConfig holds the configuration for EgressIP options.
 	// +optional
 	EgressIPConfig EgressIPConfig `json:"egressIPConfig,omitempty"`
+	GatewaySubnet  string         `json:"GatewaySubnet,omitempty"`
```

Comment:
Suggestions:
Should we put these new options inside the `GatewayConfig`?
And shouldn't we have 2 subnet options one for v4 and another for v6 ? How do you plan to have both of them in this single field?

169.254.169.0/16 isn't exactly a Gateway Subnet, if we move this into `GatewayConfig` then we could call them `V4InternalMasqueradeSubnet` and `V6InternalMasqueradeSubnet`..

Reply:
yes I think this makes more sense, will use this approach

Reply:
cc @trozet if he is ok with this.